chandra_aca/aca_image.py

```diff
@@ -1,12 +1,26 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-from itertools import count
+import os
+from math import floor
+from itertools import count, chain
 from copy import deepcopy
 from six.moves import zip
 
 import numpy as np
 from astropy.utils.compat.misc import override__dir__
 
-__all__ = ['ACAImage', 'centroid_fm']
+__all__ = ['ACAImage', 'centroid_fm', 'AcaPsfLibrary', 'EIGHT_LABELS']
+
+EIGHT_LABELS = np.array([['A1', 'B1', 'C1', 'D1', 'E1', 'F1', 'G1', 'H1'],
```

Comment:
My thought was that if we put this someplace in chandra_aca that we'd agree on a something to call them (are they labels or?) and also add the 6x6 mapping.  But I have no problem with this going in now as-is.

Reply:
Copy, sounds like a plan.